qiskit/circuit/quantumcircuit.py

```diff
@@ -1606,6 +1599,12 @@ def qasm(
     ) -> str | None:
         """Return OpenQASM string.
 
+        .. seealso::
+
+            :func:`.qasm2.dump` and :func:`.qasm2.dumps`
+                The preferred entry points to the OpenQASM 2 export capabilities.  These match the
+                interface for other serialisers in Qiskit.
```

Comment:
Why not just deprecate the `.qasm` method? Or that's done somewhere else?

Reply:
This PR is just adding the alternative now, so `QuantumCircuit.qasm` isn't eligible for deprecation yet.